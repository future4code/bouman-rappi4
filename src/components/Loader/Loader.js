import React from 'react';
import {LoaderContainer, StyledImg} from '../../style/styled';
import Logo from "../../imagens/logo-future-eats.svg";

function Loader () {
    return (
        <LoaderContainer>
            <StyledImg src={Logo} alt="deveria ter uma imagem"/>
        </LoaderContainer>
    )
}

export default Loader;