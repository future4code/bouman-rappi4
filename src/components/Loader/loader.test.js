import React from 'react'
import renderer from "react-test-renderer";
import  Loader  from './Loader';

describe ('teste componente Loader', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<Loader></Loader>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
})