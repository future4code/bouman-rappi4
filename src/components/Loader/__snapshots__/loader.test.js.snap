// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`teste componente Loader renders correctly 1`] = `
<div
  className="sc-bwzfXH ioFCEO"
>
  <img
    alt="deveria ter uma imagem"
    className="sc-htpNat fQBuIK"
    src="logo-future-eats.svg"
  />
</div>
`;
